### DISCUSSION TITLE
PyFluent unaware of Fluent licensing issues

### DISCUSSION THREAD

**Discussion by raph-luc** [ansys/pyfluent#1998] [Ideas]
Perhaps raise an Exception on PyFluent when Fluent has identified licensing issues and will therefore not work correctly?

**Comment by seanpearsonuk** [comment]
@raph-luc do you know if the way we handle this has changed/improved given the general error handling changes.

**Comment by raph-luc** [comment]
@seanpearsonuk It hasn't changed, the behavior regarding license issues is still the same as far as I can tell. 

For example, when launching Fluent with GUI but license issues occur, PyFluent receives a "successful" gRPC is serving message from Fluent, and then both Fluent and PyFluent hang indefinitely. PyFluent is completely unaware of any issues. 

In hidden mode Fluent exits on its own, and PyFluent will raise `RuntimeError: failed to connect to all addresses; last error: UNAVAILABLE: ipv6: Connection refused` on subsequent gRPC calls, also unaware of license issues.
